### PR TITLE
Changed "Linux" to "GNU"

### PR DESCRIPTION
**Pull request by TSmigielski**
Linux is just a kernel (the best one). Whilst GNU is the actual "OS", hence this makes more sense I think.

### CODE REVIEW

**Comment by DaveDavenport** [comment]
I am not sure this is clearer for most people and I suspect it might even confuse starters.
While Linux is the kernel, it is colloquially used to denoted operating systems running this kernel.
Even wikipedia says: https://en.wikipedia.org/wiki/Linux

> Linux ([/ˈlɪnʊks/](https://en.wikipedia.org/wiki/Help:IPA/English) [LIN-uuks](https://en.wikipedia.org/wiki/Help:Pronunciation_respelling_key))[[11]](https://en.wikipedia.org/wiki/Linux#cite_note-pronunciation-2-14) is a family of [open-source](https://en.wikipedia.org/wiki/Free_and_open-source_software) [Unix-like](https://en.wikipedia.org/wiki/Unix-like) [operating systems](https://en.wikipedia.org/wiki/Operating_system) based on the [Linux kernel](https://en.wikipedia.org/wiki/Linux_kernel),[[12]](https://en.wikipedia.org/wiki/Linux#cite_note-15) an

**Comment by carnager** [comment]
Even better, even Linus Torvalds says so.

**Comment by TSmigielski** [comment]
> I am not sure this is clearer for most people and I suspect it might even confuse starters.
> While Linux is the kernel, it is colloquially used to denoted operating systems running this kernel.
> Even wikipedia says: https://en.wikipedia.org/wiki/Linux
> 
> > Linux ([/ˈlɪnʊks/](https://en.wikipedia.org/wiki/Help:IPA/English) [LIN-uuks](https://en.wikipedia.org/wiki/Help:Pronunciation_respelling_key))[[11]](https://en.wikipedia.org/wiki/Linux#cite_note-pronunciation-2-14) is a family of [open-source](https://en.wikipedia.org/wiki/Free_and_open-source_software) [Unix-like](https://en.wikipedia.org/wiki/Unix-like) [operating systems](https://en.wikipedia.org/wiki/Operating_system) based on the [Linux kernel](https://en.wikipedia.org/wiki/Linux_kernel),[[12]](https://en.wikipedia.org/wiki/Linux#cite_note-15) an
> 
> 

In that case, perhaps "GNU/Linux" would be the best phrasing? It's clearer than GNU alone, and is more proper than just Linux.

File: README.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ Below is a list of the different modes:
 
 -   **combi**: Combine multiple modes into one.
 
-**Rofi** is known to work on Linux and BSD.
+**Rofi** is known to work on GNU and BSD.
```
Comment (TSmigielski) [inline README.md]:
```suggestion
**Rofi** is known to work on GNU/Linux and BSD.
```

**Comment by Karlosbubi** [comment]
Rofi is packaged for Alpine Linux too, so actually no GNU. Linux remains correct.

Edit.: It's also packaged for FreeBSD so technically just "Unixy" is correct. But I'm pretty sure Linux is fine regardless due to it's colloquial use

**Pull request closed without merging**